### PR TITLE
feat: Add mobile version

### PR DESCRIPTION
**Pull request by oyelakinG9**
- This project contains the middle section of the mobile website. it included a screenshot of the Template output below for reference. These sections of the mobile website should include:

> Works section.

> The "About myself" section (placed after the projects and before the contact form).

### CODE REVIEW

File: css/my_portfolio.css [hunk 1]
```diff
@@ -99,3 +99,224 @@ header {
   top: 23em;
   left: 60%;
 }
+
+#work-section {
```
Comment (keneogu) [inline css/my_portfolio.css:103]: - [ ] From the project specifications your work section should make use of grid layout.
`#work-section {
display: grid;
}`

File: css/my_portfolio.css [hunk 2]
```diff
@@ -99,3 +99,224 @@ header {
...
+  margin-left: 6%;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  padding-top: 7%;
+}
+
+.btn-about {
```
Comment (keneogu) [inline css/my_portfolio.css:285]: - [ ] You've done really well implementing the button styling according to the Figma template. If you look at the Figma template it gives you the styling for each interaction state of the buttons. these can be implemented in CSS using `:hover`, `:active`, `:pressed`, `:disabled`.
Figma design
btn

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @oyelakinG9 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️

- [ ] The list of project technologies  (e.g., "JavasScript", "HTML" and "CSS") are not centered please fix it.
your design
cen
Figma design
cen1

- [ ] You have done a very good job on the project but I suggest you the same font size used on the template
your design
font
Figma 
font1
design


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

**Comment by oyelakinG9** [comment]
Hi keneogu'
I'll appreciate your comments.
Actually, I don't know Hover is part of the requirement, I've checked those areas and modified them as per PR requirement. You can check out for new updates.
Thanks

File: css/my_portfolio.css [hunk 3]
```diff
@@ -99,3 +99,249 @@ header {
...
+}
+
+.card-background-image {
+  background-image: url("../assets/images/Img_Placeholder2.png");
+  height: 24em;
+  width: 92%;
+  margin: 8% 4%;
+}
+
+.card-headline {
```
Comment (t-yanick) [inline css/my_portfolio.css:216]: - [ ] Hi there, you have really done so well, but I think you still have not applied the right fonts. Please click on the text in the Figma design and you will see the applied properties to the left. Check screenshot below of your design on the left and [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) on the right. 
- [ ] Kindly take a closer look at your design and the Figma design and fix the necessary fonts
image

File: index.html [hunk 4]
```diff
@@ -1,67 +1,271 @@
...
+          <img src="assets/images/Vector.png" alt="" />
+        </li>
+      </ul>
+    </div>
+    <div class="group1 pos-absolute d-none-lg"></div>
+    <!-- <div class="group2 pos-absolute d-none d-block-lg"></div> -->
+  </section>
+
+  <!-- work section -->
+  <section id="work-section" class="work-section">
```
Comment (t-yanick) [inline index.html:68]: - [ ] As indicated by the previous reviewer, the work section is supposed to have a `display: grid` and looking at your CSS in `lines 30 to 32`, you defined a `d-grid` class there, which is good CSS practice. I think you need to add this class in your index.html line 68.

File: index.html [hunk 5]
```diff
@@ -1,67 +1,271 @@
...
+          <p class="card-headline-text">A daily selection of privately personalized reads; no accounts or sign-ups
+            required. has been the industry's standard</p>
+        </div>
+        <div class="card-skills">
+          <ul class="skills d-flex list-style-none">
+            <li class="skill2 skillcard">html</li>
+            <li class="skill3 skillcard">bootstrap</li>
+            <li class="skill4 skillcard">Ruby</li>
+          </ul>
+          <button class="card-button btn-cards">See Project</button>
```
Comment (t-yanick) [inline index.html:110]: - [ ] Kindly review the Hover of the `See project` button to make sure it is working as per the screenshot below. When I Hover over the button, I don't have the same experience as per Figma design. In the Hover of this button, apply a `background: #FFF5E1;`

image

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @oyelakinG9 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.




 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by moise10r — APPROVED** [review]
Hi @oyelakinG9  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
121368858-ff93ee80-c900-11eb-9ee8-e43b2b0d3382


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/my_portfolio.css (hunk 3 above)
Comment (oyelakinG9) [reply css/my_portfolio.css:216]: The font: "Crete Round" was rejected by linter; That's why I used font-family: "Crete Round", sans-serif"

**Pull request merged**